src/main.c

```diff
@@ -33,6 +33,56 @@ static void init_default_values()
 }
 
 static char *submitter = NULL;
+#define LOG_NAME_PATH_DEFAULT "D:\\kernel_tests"
+#define LOG_FILE_EXT ".log"
+static const char* name_log_format = LOG_NAME_PATH_DEFAULT "-%s" LOG_FILE_EXT;
+static char* name_log = NULL;
+
+void load_name_file(const char* file_path)
+{
+    HANDLE handle = CreateFile(
+        file_path,
+        GENERIC_READ,
+        FILE_SHARE_READ,
+        0,
+        OPEN_EXISTING,
+        FILE_ATTRIBUTE_NORMAL,
+        NULL
+    );
+    if (handle == INVALID_HANDLE_VALUE) {
+        return;
+    }
+
+    DWORD file_size = GetFileSize(handle, NULL);
+    if (file_size == INVALID_FILE_SIZE) {
+        return;
+    }
+
+    char* buffer = (char *)malloc(file_size);
+    if (buffer == NULL) {
+        return;
+    }
+
+    DWORD bytes_read = 0;
+    BOOL result = ReadFile(handle, buffer, file_size, &bytes_read, NULL);
+    if (result == 0 || bytes_read != file_size) {
+        return;
+    }
+
+    CloseHandle(handle);
+
+    char *line;
+    char *rest = buffer;
+    if ((line = strtok_r(rest, NEWLINE_DELIMITER, &rest))) {
+        size_t length = strlen(line);
+        if (length) {
+            name_log = calloc(length + 1, sizeof(char));
```

Comment:
Best not access a global variable here. Instead make `name_log` a return value

Reply:
Technically, `name_log` is bind to only main.c file hence the `static` assigned. From my view, it is a local (global) variable just like submitter, tests_to_run, and tests_exclude variables do.

Who knows, maybe there will be additional feature if there's a request for it. 🤷‍♂️